package.json

```diff
@@ -139,6 +139,7 @@
     "ts-loader": "^3.5.0",
     "ts-node": "^7.0.1",
     "tslint-config-airbnb": "^5.11.1",
+    "typemoq": "^2.1.0",
```

Comment:
Sorry to have to do this but I don't think we'll be allowed to introduce a new testing dependency in the mobile app :(

Reply:
Shame, it can save a lot of space when setting up test data! Anyway, it's gone. 💥